packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.test.tsx

```diff
@@ -312,6 +312,14 @@ describe('DetailsList', () => {
       );
     };
 
+    const getCellValueKey = (item: any, index: number, column: IColumn) => {
+      let valueKey = item && column && column.fieldName ? item[column.fieldName] : '';
+      if (valueKey === null || valueKey === undefined) {
```

Comment:
Perhaps also verify `valueKey` is not an empty string else it will log warnings for duplicate keys.